apps/src/templates/teacherHomepage/TeacherHomepage.story.jsx

```diff
@@ -1,42 +1,110 @@
 import React from 'react';
 import TeacherHomepage from './TeacherHomepage';
 
+const announcements = [
+  {
+    heading: "Go beyond an Hour of Code",
+    buttonText: "Go Beyond",
+    description: "Go Beyond an Hour of Code and explore computer science concepts with your students every week. Code.org offers curriculum, lesson plans, high quality professional learning programs, and tons of great tools for all grade levels - and it's free. No experience required - find the next step that's right for your classroom.",
```

Comment:
Such as? Real announcement descriptions will be coming in as a loc'ed string with data_t_suffix.